srml/chainrelay/bridge/ethereum/src/lib.rs

```diff
@@ -17,7 +17,7 @@ use system::ensure_signed;
 
 pub trait Trait: system::Trait {
 	type Event: From<Event<Self>> + Into<<Self as system::Trait>::Event>;
-	type Ring: LockableCurrency<Self::AccountId, Moment = Self::BlockNumber>;
+	type Ring: LockableCurrency<Self::AccountId, Moment = staking::Timestamp>;
```

Comment:
hmm.... staking::Timestamp? 
The dependency on staking module does not looks good.